// vitepress/dist/node-cjs/index.cjs
'use strict';

var init = require('./serve-680123a9.cjs');
var vite = require('vite');
require('dns');
require('path');
require('readline');
require('os');
require('tty');
require('assert');
require('events');
require('stream');
require('buffer');
require('url');
require('fs');
require('crypto');
require('shiki');
require('child_process');
require('module');
require('util');
require('net');
require('zlib');
require('http');
require('querystring');
require('node:process');
require('node:readline');
require('node:tty');
require('constants');



exports.ScaffoldThemeType = init.ScaffoldThemeType;
exports.build = init.build;
exports.createMarkdownRenderer = init.createMarkdownRenderer;
exports.createServer = init.createServer;
exports.defineConfig = init.defineConfig;
exports.defineConfigWithTheme = init.defineConfigWithTheme;
exports.defineLoader = init.defineLoader;
exports.init = init.init;
exports.resolveConfig = init.resolveConfig;
exports.resolvePages = init.resolvePages;
exports.resolveSiteData = init.resolveSiteData;
exports.scaffold = init.scaffold;
exports.serve = init.serve;
Object.defineProperty(exports, 'loadEnv', {
  enumerable: true,
  get: function () { return vite.loadEnv; }
});
